packages/kolibri-common/components/ExtraDemographics/index.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+
+  <div>
+    <ExtraDemographicField
+      v-for="field in customSchema"
+      :key="field.id"
+      :field="field"
+      :value="(value || {})[field.id] || ''"
+      :disabled="disabled"
+      @select="v => setInput(field.id, v)"
+    />
+  </div>
+
+</template>
+
+
+<script>
+
+  import isPlainObject from 'lodash/isPlainObject';
+  import isNull from 'lodash/isNull';
+  import ExtraDemographicField from './ExtraDemographicField';
+
+  export default {
+    name: 'ExtraDemographics',
+    components: {
+      ExtraDemographicField,
+    },
+    props: {
+      value: {
+        required: true,
+        validator: o => isNull(o) || isPlainObject(o),
+      },
+      facilityDatasetExtraFields: {
+        type: Object,
+        required: true,
+      },
+      disabled: {
+        type: Boolean,
+        default: false,
+      },
+    },
+    computed: {
+      customSchema() {
+        return this.facilityDatasetExtraFields.demographic_fields || [];
+      },
+    },
+    methods: {
+      setInput(key, value) {
```

Comment:
Non-blocking nitpick - it took me a couple reads of this to recognize `this.value` and `value` where indeed completely different things here 😅 -- it just read a little weird because of the duplicated name so might be helpful to call the param `newVal` or something